package.json

```diff
@@ -21,6 +21,7 @@
     "io-ts": "^2.2.13",
     "lodash": "^4.17.20",
     "monocle-ts": "^2.3.11",
+    "plotly.js": "^2.5.1",
```

Comment:
Are we 100% sure we need this?  I'm going to try without...

Reply:
I'm stuck too!

If I do `yarn remove plotly.js` then yarn.lock seems to have entries for v2.6.0 but the all important `node_modules/plotly.js/package.json` still says `"version": "1.58.5"`

I tried deleting the node_modules directory and rebuilding but it's the same.

Very strange - need help please @dmfalke @jtlong3rd

Reply:
@bobular First of all, hank you for your approval and merge at web-components PR - I just checked emails. As for the plotly.js entry, yes it probably needs to have it at dependencies as it did not automatically upgrade with the web-components. That was the reason why I added it in there. This issue arose sometime last week when Connor noticed this. Such an approach was my solution to resolve.

Reply:
I believe the issue last week was to do with using `yarn link`. That will not necessarily trigger a transitive dependency to be upgraded.

Running `yarn upgrade <packageName>` should cause transitive dependencies to be updated. I will investigate further soon.

Reply:
I was able to resolve this by deleting `yarn.lock` and running `yarn install` again. I just pushed this change.